source/index.html.erb.md

```diff
@@ -3,11 +3,11 @@ title: "Quick Start Tips"
 status: editors-draft
 ---
 
-These guides provide you with practical tips to get started with accessibility and pointers to detailed resources to help you learn more.
+These pages provide you with practical tips to get started with web accessibility, and pointers to resources to learn more.
 
-* [Design tips](design.html): For visual or interface design.
-* [Development tips](development.html): For front-end website development.
-* [Content tips](content.html): For writing content for the web.
-* [Evaluation tips](evaluation.html): For website evaluation process.
-* [Project management tips](project_management.html): For create processes supportive of accessibility. 
-* [Advocacy tips](advocacy.html): For championing accessibility in organizations.
\ No newline at end of file
+* [Design tips](design.html): For visual design and interface design.
+* [Development tips](development.html): For front-end coding.
+* [Content tips](content.html): For writing content.
+* [Evaluation tips](evaluation.html): For evaluating websites.
```

Comment:
Actually this is more particularly about the evaluation process. Easy Checks is more targeted towards evaluating websites and the view was not to replicate that resource.